src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,222 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String createQuery = "INSERT INTO cars (model, registration_number, manufacturer_id) "
+                + "VALUES (?, ?, ?)";
+        boolean isCreated = false;
```

Comment:
we don't need this variable, if it's not created we probably will get exception and code will stop its' execution

Reply:
I agree. I was very reinsured in both cases)))